tests/molecules/LiH_dimer_pp/det_vmc_hf_sdj_xml.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<simulation>
+  <project id="det_vmc_hf_sdj_xml" series="0"/>
+  <random seed="9124"/>
```

Comment:
Use a different random seed for each input file.  Note that seeds are identical if they differ by multiples of 1024 at present.

Reply:
What is the purpose of having different seeds?
I'm actually thinking having the same seed is better. If xml input and h5 inputs are identical, we can only use and maintain the same gold data.

Reply:
Good point on the gold data. In this case with consistent seeds we would check the XML and HDF5 handling.

Different seeds for testing different functionality adds a bit more entropy to the testing.

Reply:
The "gold" input file will never change.  Only when creating gold input files for different tests would we use differing seeds, for the entropy reason Paul noted.  When the reference data needs regenerating, the seed will not change.